readme.md

```diff
@@ -51,6 +52,7 @@ Options:
   -h --help      Shows this help message
   -a --all       Includes pre and post scripts on the list
   -m --multiple  Allows a selection of multiple tasks to run at once
+	-s --size      Sets the amount of tasks to show in the list without scrolling
```

Comment:
oops, this has a tab instead of spaces here

Reply:
Oppsi.. Fixed that.